awx/main/tests/functional/conftest.py

```diff
@@ -252,7 +252,7 @@ def credentialtype_insights():
     return insights_type
 
 
-@pytest.fixture
+@pytest.yield_fixture
```

Comment:
Neat, I've never seen this before 👍

Reply:
this use is deprecated?

https://docs.pytest.org/en/latest/yieldfixture.html

Reply:
oh, nice catch - looks like they've updated it so this behavior is automatic now. I'll switch this back to the regular decorator.